util/s3/fileIterator.go

```diff
@@ -26,17 +26,19 @@ import (
 type fileIterator struct {
 	filePaths []string
 	bucket    string
+	subFolder string
 	next      struct {
 		path string
 		f    *os.File
 	}
 	err error
 }
 
-func makeFileIterator(files []string, bucket string) s3manager.BatchUploadIterator {
+func makeFileIterator(files []string, bucket string, subFolder string) s3manager.BatchUploadIterator {
```

Comment:
The other function named this `targetFolder`, might as well keep them consistent.